age-app/src/app/filling-container.tsx

```diff
@@ -0,0 +1,158 @@
+"use client";
+import React from "react";
+import { useState } from "react";
+import "./filling-container.css";
+import FillingBox from "./filling-box";
+export default function FillingContainer() {
+  //store input day, month, year
+  const [day, setDay] = useState("");
+  const [month, setMonth] = useState("");
+  const [year, setYear] = useState("");
+//store the calculation of days, month, year
+  const [resultDay, setResultDay] = useState("--");
+  const [resultMonth, setResultMonth] = useState("--");
+  const [resultYear, setResultYear] = useState("--");
+//store error msg for day, month, year
+  const [errorDayMsg, setErrorDayMsg] = useState("");
+  const [errorMonthMsg, setErrorMonthMsg] = useState("");
+  const [errorYearMsg, setErrorYearMsg] = useState("");
+   //this is for error red effect
+  const [errorInvalid, setErrorInvalid] = useState("");
+//get current date
```

Comment:
Comments should not describe what the code does, they should describe why it does what it does.